sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/keyvault/certificates/AzureKeyVaultCertificateAutoConfiguration.java

```diff
@@ -21,13 +22,14 @@
 import org.springframework.context.annotation.Bean;
 
 /**
- * Auto-configuration for a {@link CertificateClientBuilder} and Azure Key Vault secret clients.
+ * {@link EnableAutoConfiguration Auto-configuration} for Azure Key Vault Certificate support.
+ *
+ * @since 4.0.0
  */
 @ConditionalOnClass(CertificateClientBuilder.class)
 @ConditionalOnExpression("${spring.cloud.azure.keyvault.certificate.enabled:true}")
 @ConditionalOnProperty("spring.cloud.azure.keyvault.certificate.endpoint")
-public class AzureKeyVaultCertificateAutoConfiguration extends AzureServiceConfigurationBase {
-
+public final class AzureKeyVaultCertificateAutoConfiguration extends AzureServiceConfigurationBase {
```

Comment:
There's no need to mark the class final here.

Reply:
It won't cause any problem if we add `final`, how about keep it?

Reply:
I think we've chosen to follow Spring Boot's style, and to be consistent for all our classes in this autoconfigure module, I prefer not adding the final here.